### ISSUE TITLE
Best accuracy?

### ISSUE THREAD

**Issue by ahmed-shariff** [JorgeCeja/food-101#1]
What is the best accuracy you got from training on the food 101 dataset?

**Comment by JorgeCeja** [comment]
Hey @ahmed-shariff I am in the process of getting a better GPU in order to train the model for more epochs. It currently takes too long to train and I am trying to optimizing the training process and get a hold of better hardware. I will report back once I get a more definite answer.

**Comment by JorgeCeja** [comment]
Hello @ahmed-shariff the best accuracy and have gotten is 79% with tweaked hyperparameters. I will be posting the changes and the weights as soon as my other test finish. I also fixed the issue I was having with the training time, now the model trains much faster.
